File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/VirksomhetEksportPerKvartal.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon;

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene.Orgnr;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domene.ÅrstallOgKvartal;

public class VirksomhetEksportPerKvartal {
  private final Orgnr orgnr;
  private final ÅrstallOgKvartal årstallOgKvartal;
  private final boolean eksportert;

  public String getOrgnr() {
    return orgnr.getVerdi();
  }

  public int getÅrstall() {
    return årstallOgKvartal.getÅrstall();
  }

  public int getKvartal() {
    return årstallOgKvartal.getKvartal();
  }

  public ÅrstallOgKvartal getÅrstallOgKvartal() {
    return årstallOgKvartal;
  }

  public boolean eksportert() {
    return eksportert;
  }

  public VirksomhetEksportPerKvartal(
      Orgnr orgnr, ÅrstallOgKvartal årstallOgKvartal, boolean eksportert) {
    this.orgnr = orgnr;
    this.årstallOgKvartal = årstallOgKvartal;
    this.eksportert = eksportert;
  }
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/eksportAvSykefraværsstatistikk/domene/VirksomhetEksportPerKvartal.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.eksportAvSykefraværsstatistikk.domene

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene.Orgnr
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene.ÅrstallOgKvartal

class VirksomhetEksportPerKvartal(
    private val orgnr: Orgnr, val ÅrstallOgKvartal: ÅrstallOgKvartal, private val eksportert: Boolean
) {
    fun getOrgnr(): String {
        return orgnr.verdi
    }

    val Årstall: Int
        get() = ÅrstallOgKvartal.årstall
    val kvartal: Int
        get() = ÅrstallOgKvartal.kvartal

    fun eksportert(): Boolean {
        return eksportert
    }
}
